### PR TITLE
44 utrack2018b

### PR DESCRIPTION
**Pull request by jgesell**
Lost track of branch I was on, so most of the images have been uploaded to branch 44 by mistake.

### CODE REVIEW

File: bcl2fastq2.17/2.0.0/unittest.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 commands:
-  - cmd: '-c "bcl2fastq --version 2>&1| head -n2 | tail -n1"'
+  - cmd: '-c "cellranger--version" | head -n3'
```
Comment (vsmalladi) [inline bcl2fastq2.17/2.0.0/unittest.yml:2]: Wrong command

**Review by vsmalladi — CHANGES_REQUESTED** [review]
Some test seems to not use the correct command.

**Comment by jgesell** [comment]
That is due to some of the older images using the entrypoint flag.

**Review by jgesell — COMMENTED** [review]
Verified other changes, and ensured that those with entrypoints are done correctly.

**Pull request merged**